packages/manager/modules/pci/src/projects/project/storages/objects/object/object.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.project.storages.objects.object', {
+      url: '/{containerId}',
+      component: 'pciProjectStorageContainersContainer',
+      resolve: {
+        containerId: /* @ngInject */ $transition$ => $transition$.params().containerId,
+        addObject: /* @ngInject */ ($state, projectId, containerId) => () => $state.go('pci.projects.project.storages.objects.object.add', {
```

Comment:
Looks like this won't add an object but this will change state to be able to add an object, this is confusing